pkg/operator/controller/gatewayclass/subscription.go

```diff
@@ -16,6 +17,11 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+var (
+	serviceMeshOperatorDesiredVersion = "servicemeshoperator.v2.6.2"
```

Comment:
I'm not sure this is going to work.  If we change the version in a future release and need to backport a different change in the same file, won't this get overwritten?  I think normally version information is recorded in a manifest yaml.  Can you find a way to do this without hardcoding the version?

Reply:
I'm not too worried about merge conflicts, but I wouldn't object to putting the version string in a separate `.go` file, or putting it in a yaml manifest, maybe as an environment variable, à la `WATCH_NAMESPACE` etc., which would also make it easier to override for testing and development.  If we do define it in Go code, it should be a `const`.